app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-form-tracker.js

```diff
@@ -123,5 +123,12 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     return answers
   }
 
+  Ga4FormTracker.prototype.useFallbackValue = function (formElement) {
+    if (formElement.closest('[data-ga4-form-undefined-if-text-is-empty]')) {
```

Comment:
Maybe `data-ga4-form-no-answer-undefined`? It's still a bit of a mouthful though.

Reply:
Thanks, i'll use this instead